pages/docs/installation/node.mdx

```diff
@@ -40,10 +46,17 @@ nvm use v12 # use v12 of node.js
 sudo apt install redis-server # Install Redis
 
 sudo apt install postgresql postgresql-contrib # Install Postgres
+```
+
+### Create Grouparoo Database
+
+Using either Postgres or SQLite, create a database for Grouparoo to use. This is where the application data will be stored, such as profiles, apps, and sources. In our example, we've called the database `grouparoo_development`. This is what it looks like for Postgres:
```

Comment:
In SQLite, you don't "create" a database like you need to with postgres.  There is no `createdb` equivalent in SQLlite